app/models/message.rb

```diff
@@ -19,13 +21,6 @@ class Message < ApplicationRecord
 
   after_create_commit :broadcast_message
 
-  def for_openai
-    {
-      role: role,
-      content: content_text
-    }
-  end
-
```

Comment:
Now that we support images, we need to consider the assistant being used to know how to properly render this so I moved it into the `open_a_i.rb` client